apps/admin-gui/src/app/users/pages/user-detail-page/user-dashboard/dashboard-card/dashboard-card.component.html

```diff
@@ -13,15 +13,17 @@
         [pageSizeOptions]="null"
         [displayedColumns]="['id', 'recent', 'shortName', 'name']"
         [recentIds]="this.recentIds"
-        [vos]="this.objects">
+        [vos]="this.objects"
+        [selection]="selected">
```

Comment:
Since there isn't displayed column with checkboxes on the dashboard page, you don't have to put selection as a parameter here -> the same for all other lists here ... IMHO you can restore changes in the whole file and also in the ts file for the dashboard and it should still work well.
```suggestion
        [vos]="this.objects">
```